### PR TITLE
Brmp 12118 dev

### PR DESCRIPTION
**Pull request by pramodganore**

### CODE REVIEW

**Review by chinchalinchin — APPROVED** [review]
assuming the user provided service on calc dev has been updated with the new values, I think it looks good to me.

File: calc/context_processors.py [hunk 1]
```diff
@@ -13,6 +13,13 @@ def google_analytics_tracking_id(request):
     '''Include GA_TRACKING_ID in all request contexts'''
     return {'GA_TRACKING_ID': settings.GA_TRACKING_ID}
 
+def google_analytics_tracking_id_dx(request):
+    '''Include GA_TRACKING_ID in all request contexts'''
+    return {'GA_TRACKING_ID_DX': settings.GA_TRACKING_ID_DX}
+
+def google_analytics_tracking_id_dx(request):
```
Comment (chinchalinchin) [inline calc/context_processors.py:20]: This function has the same name as the previous function. it's causing a build error on circleci.

**Review by chinchalinchin — CHANGES_REQUESTED** [review]
need to change function name in context_processors.py

File: calc/context_processors.py [hunk 2]
```diff
@@ -13,6 +13,13 @@ def google_analytics_tracking_id(request):
     '''Include GA_TRACKING_ID in all request contexts'''
     return {'GA_TRACKING_ID': settings.GA_TRACKING_ID}
```
Comment (chinchalinchin) [inline calc/context_processors.py:15]: the linters on circleci want two blank lines between functions

File: calc/context_processors.py [hunk 3]
```diff
@@ -13,6 +13,13 @@ def google_analytics_tracking_id(request):
...
 
+def google_analytics_tracking_id_dx(request):
+    '''Include GA_TRACKING_ID in all request contexts'''
+    return {'GA_TRACKING_ID_DX': settings.GA_TRACKING_ID_DX}
+
+def google_analytics_tracking_id_dx(request):
+    '''Include GA_TRACKING_ID in all request contexts'''
+    return {'GTM_TRACKING_ID': settings.GTM_TRACKING_ID}
```
Comment (chinchalinchin) [inline calc/context_processors.py:23]: the linters on circleci want two blank lines between functions. See build results herE: [build failure](https://app.circleci.com/pipelines/github/GSA/calc/963/workflows/af711c47-eac1-47e1-8a1e-7c535e1f4892/jobs/4202)

File: calc/context_processors.py (hunk 1 above)
Comment (pramodganore) [reply calc/context_processors.py:20]: i had the _dx portion

**Review by dsomalaraju — APPROVED** [review]

**Review by theodesta — APPROVED** [review]

File: calc/context_processors.py (hunk 2 above)
Comment (pramodganore) [reply calc/context_processors.py:15]: updated

File: calc/context_processors.py (hunk 1 above)
Comment (pramodganore) [reply calc/context_processors.py]: updated

**Review by chinchalinchin — APPROVED** [review]

**Pull request merged**